client-cli/src/command/transaction_command.rs

```diff
@@ -0,0 +1,119 @@
+use std::str::FromStr;
+
+use failure::ResultExt;
+use hex::decode;
+use quest::{ask, choose, text, yesno};
+use structopt::StructOpt;
+
+use chain_core::common::H256;
+use chain_core::init::address::RedeemAddress;
+use chain_core::init::coin::Coin;
+use chain_core::tx::data::address::ExtendedAddr;
+use chain_core::tx::data::attribute::TxAttributes;
+use chain_core::tx::data::output::TxOut;
+use client_common::{ErrorKind, Result};
+use client_core::WalletClient;
+
+use crate::ask_passphrase;
+
+#[derive(Debug, StructOpt)]
+pub enum TransactionCommand {
```

Comment:
It's almost copy-pasted from https://github.com/crypto-com/chain/blob/master/signer-cli/src/commands/transaction_command.rs

Is there any use case for signer-* ? Would it make sense to remove signer-* and move that functionality into client-* crates (if any)?

Reply:
`signer-*` should only be removed once we have multisig support in `client`.